composer.json

```diff
@@ -17,7 +17,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.0",
-        "psr/log": "^3.0",
+        "psr/log": "^1.1@dev",
```

Comment:
This dependency should not be `@dev`

Reply:
Je l'ai modifié sur le dernier commit [`ea0e706` (#5)](https://github.com/php-etl/pipeline/pull/5/commits/ea0e70628a35c092107f9e88e7e0b67f6ce67961)